templates/example.gotxt

```diff
@@ -2,7 +2,11 @@
 
 {{- template "doc" .Doc -}}
 
-{{- codeBlock "go" .Code -}}
+{{- codeBlock "go" .Code }}
+
+Output:
```

Comment:
I generally avoid including bare text in the templates because it creates formatting issues, especially when conditionals, loops, etc. get involved. I think this will end up working better in the long run if we use one of the constructs like `header`. It takes care of all of the surrounding whitespace and whatnot for you, so the template itself can read a bit more like code.

For this, I'd probably just do a hardcoded level 4 header or something like that. Very minimal styling/emphasis:

```
{{- header 4 "Output" -}}
```

Reply:
Implemented in latest commit